ledger-api/grpc-definitions/com/digitalasset/participant/admin/api/v1/participant_management_service.proto

```diff
@@ -0,0 +1,54 @@
+
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.digitalasset.admin.api.v1;
+
+option java_outer_classname = "PartyManagementServiceOuterClass";
+option java_package = "com.digitalasset.admin.api.v1";
+
+// Inspect the administrative state of a ledger participant and modify the
+// parts that are modifiable. We use 'backing participant' to refer to this
+// specific participant in the methods of this API.
+service ParticipantManagementService {
+
+  // Return the identifier of the backing participant.
+  rpc GetParticipantId (GetParticipantIdRequest) returns (GetParticipantIdResponse)
+
+  // List the participants known by the backing participant.
+  rpc ListKnownParticipants (ListKnownParticipantsRequest) returns (ListKnownParticipantsResponse)
```

Comment:
I don't think that this information makes sense here. What a participant is, is very specific to the implementation of `x`. I wouldn't leak that into this API here.